source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -0,0 +1,185 @@
+positional ``$`` operator. 
+
+The positional ``$`` operator specifies the first element of the array
+matched by your query filter.  
+
+Example
+```````
+
+The following example performs these actions:
+
+- Query for the position in ``qty`` where the value is "18"
+- Decrement the value by "3"  using the ``inc()`` method of the ``Updates`` builder  
+- Match and update the first element in the array with a positional ``$`` operator 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/UpdateArray.java
+   :language: java
+   :dedent:
+   :start-after: begin updateValueExample
+   :end-before: end updateValueExample
+
+The example above updates the original document to the following state:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "color": "green", "qty": [8, 12, 15] }
+
+For additional information on the operator and method mentioned in this
+section, see the following resources: 
+
+- :manual:`Positional $ Operator </reference/operator/update/positional>` Server Manual Entry
+- :java-core-api:`inc() <com/mongodb/client/model/Updates.html#inc(java.lang.String,java.lang.Number)>` API Documentation
+
+All Array Elements
+~~~~~~~~~~~~~~~~~~
+
+To update all elements in an array that match your query filter, use the
+all positional ``$[]`` operator. 
+
+Example
+```````
+
+The following example performs these actions: 
+
+- Query for the sample document
+- Multiplies each value by "2"  using the ``mul()`` method of the ``Updates`` builder 
+- Updates all elements in the ``qty`` array using an all positional operator
+
+.. literalinclude:: /includes/fundamentals/code-snippets/UpdateArray.java
+   :language: java
+   :dedent:
+   :start-after: begin updateAllElementsExample
+   :end-before: end updateAllElementsExample
+
+The example above updates the original document to the following state:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "color": "green", "qty": [16, 24, 36] }
+
+For additional information on the operator and method mentioned in this
+section, see the following resources: 
+
+- :manual:`Positional $[] Operator </reference/operator/update/positional-all/>` Server Manual Entry
+- :java-core-api:`mul() <com/mongodb/client/model/Updates.html#mul(java.lang.String,java.lang.Number)>` API Documentation
+
+Multiple Array Elements
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To update multiple array elements that match your query filter, use the
+filtered positional ``$[<identifier>]`` operator. You must include an
+array filter to specify which array elements to update.
+
+The ``<identifier>`` is the placeholder value you give to represent
+elements of the array field. This value must begin with a
+lowercase letter and contain only alphanumeric characters.
+
+Example
+```````
+
+The following example performs these actions:
+
+- Query for the sample document
+- Set a ``FindOneAndUpdateOptions.arrayFilters`` condition to search for values less than "15"
```

Comment:
Rephrased to: Set the array filter to search for values less than "15"

I believe this is simpler and to the point. The extra details you have seem to be unnecessary since they're mentioned prior to the example.